src/core_atmosphere/Registry.xml

```diff
@@ -1348,6 +1379,10 @@
                              description="Graupel mixing ratio"
                              packages="mp_thompson_in;mp_wsm6_in"/>
 
+                        <var name="nc" array_group="number" units="nb kg^{-1}"
+                             description="Cloud water number concentration"
+                             packages="bl_mynn_in"/>
+
```

Comment:
This looks like something that is part of a different parameterization and not Noah. Can you verify this?